modules/security-context-constraints-about.adoc

```diff
@@ -35,10 +35,17 @@ The cluster contains several default security context constraints (SCCs) as desc
 
 [IMPORTANT]
 ====
-Do not modify the default SCCs. Customizing the default SCCs can lead to issues when some of the platform pods deploy or {product-title} is upgraded. During upgrades between some versions of {product-title}, the values of the default SCCs are reset to the default values, which discards all customizations to those SCCs.
```

Comment:
> During upgrades between some versions of {product-title}, the values of the default SCCs are reset to the default values, which discards all customizations to those SCCs.

I would like to keep this original statement, which was introduced in https://github.com/openshift/openshift-docs/pull/33416 . I do not see your change looks better.